sips/sip-022/sip-022_generalized_marketplace_standard.md

```diff
@@ -0,0 +1,310 @@
+# Preamble
+
+SIP Number: 022
+
+Title: The Generalized Standard for non-custodial Marketplace traits
+
+Authors: werner.btc (werner at stx.fan), Friedger, Mike, Jason and Jamil.
+
+Consideration: Technical
+
+Type: Standard
+
+Status: draft
+
+Created: 10 January 2023
+
+License: CC0-1.0
+
+Sign-off: Brice Dobry <brice@hiro.so>, Technical Consideration Advisory Boards (CAB) Chair
+Sign-off: Jude Nelson <jude@stacks.org>, Technical Steering Committee Chair
+
```

Comment:
Should add Traits field.
Traits: `marketplace-functions` and `commission-functions`